docs/core/diagnostics/diagnostic-libraries.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Diagnostic libraries
+description: An overview of the available diagnostic libraries in .NET.
+ms.date: 10/16/2023
+ms.topic: overview
+---
+
+# Diagnostic libraries
+
+Available as a set of .NET Extensions, various diagnostic libraries provide a set of APIs and tools that enable you to collect and analyze diagnostic information about your .NET applications. From summarizing exceptions, to determining resource utilization and monitoring, to exposing health checks, and providing probes for Kubernetes, the diagnostic packages provide a rich set of functionality that can be used to instrument your apps.
+
+The following NuGet packages are available:
+
+| NuGet package | Description |
+|--|--|
+| [📦 Microsoft.Extensions.Diagnostics.ExceptionSummarization](https://www.nuget.org/packages/Microsoft.Extensions.Diagnostics.ExceptionSummarization) | Provides a mechanism for retrieving exception summary information. |
```

Comment:
I would drop 'Provides a' || 'Provides' prefixes to make it shorter and less repetetive.